### ISSUE TITLE
Consider publishing helm chart artifacts to Github

### ISSUE THREAD

**Issue by caugustus-sourcegraph** [sourcegraph/sourcegraph#36832]

**Comment by sourcegraph-bot-2** [comment]
Heads up @sourcegraph/delivery - the "team/delivery" label was applied to this issue.

**Comment by briemcnally** [comment]
@kevinwojo given https://github.com/sourcegraph/sourcegraph/issues/36833 can we close this?

**Comment by kevinwojo** [comment]
@briemcnally no, this is still a valid issue. Crystal and I happened to put in the same issue as we were independently looking at the same issue. This should stay open until we decide the next steps.
